src/js/components/Box/__tests__/Box-test.js

```diff
@@ -94,6 +94,9 @@ describe('Box', () => {
         <Box alignContent="around" />
         <Box alignContent="stretch" />
         <Box alignContent="end" />
+        <Box alignContent="around" />
+        <Box alignContent="baseline" />
+        <Box alignContent="evenly" />
```

Comment:
If we are also going to support the caller passing valid CSS string values, should we also incorporate tests for those values as well (i.e. in addition to the mapped values)?